includes/class-apple-news.php

```diff
@@ -1,105 +1,184 @@
 <?php
 /**
+ * Publish to Apple News Includes: Apple_News class
+ *
+ * Contains a class which is used to manage the Publish to Apple News plugin.
+ *
+ * @package Apple_News
+ * @since 0.2.0
+ */
+
+/**
  * Base plugin class with core plugin information and shared functionality
  * between frontend and backend plugin classes.
  *
- * @author  Federico Ramirez
- * @since   0.2.0
+ * @author Federico Ramirez
+ * @since 0.2.0
```

Comment:
The latest push rolls back the change from `public static` to `const` and restores variable names for each.